### PR TITLE
Hookup Announcements with API + postedAt DateTime

### PR DESCRIPTION
**Pull request by ahsu1230**
@MiniViolin @mythicat554 

Please take a look sometime and review my code. If there's anything that looks weird or you might have a question, do post something! Don't hold back!
This change does the following:

- Hooks up the announcement page with API & Modals
- Moves around styling to display the rows in a neat way.
- Introduces a postedAt field that will assumes we are posting immediately and successfully sends to backend to be saved.

- In the near future, we will use a DateTime picker to allow a user to easily select what day and what time to post their announcement.
- We could consider a front-end utility class to help us with DateTime stuff (since the Javascript DateTime has so many options).

If it all looks good, please approve this PR. Thanks!

### CODE REVIEW

File: orion/pkg/domains/announce.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ type Announce struct {
 	CreatedAt time.Time    `db:"created_at"`
 	UpdatedAt time.Time    `db:"updated_at"`
 	DeletedAt sql.NullTime `db:"deleted_at"`
-	PostedAt  time.Time    `db:"posted_at"`
+	PostedAt  time.Time    `db:"posted_at" json:"postedAt"`
```
Comment (ahsu1230) [inline orion/pkg/domains/announce.go]: backend change

File: orion/sites/admin/src/static.styl [hunk 2]
```diff
@@ -46,6 +46,9 @@ spacing20x = (spacing1x * 20)
 spacing24x = (spacing1x * 24)
 spacing30x = (spacing1x * 30)
 spacing32x = (spacing1x * 32)
+spacing36x = (spacing1x * 36)
+spacing42x = (spacing1x * 42)
+spacing48x = (spacing1x * 48)
```
Comment (ahsu1230) [inline orion/sites/admin/src/static.styl]: sometimes I wonder if all these are necessary.

File: orion/sites/admin/src/announce/announceEdit.js [hunk 3]
```diff
@@ -55,3 +153,8 @@ export class AnnounceEditPage extends React.Component {
     );
   }
 }
+
+function createDate() {
+  let now = new Date(Date.now());
+  return now;
```
Comment (ahsu1230) [inline orion/sites/admin/src/announce/announceEdit.js]: add a TODO. We are going to use a DateTime selector to figure out how to select a certain day and at what time to post an announcement.

**Comment by MiniViolin** [comment]
This looks pretty good to me, I don't see anything strange.

**Pull request merged**